### ISSUE TITLE
При вызове настроек нет возможности вернуться к списку найденного

### ISSUE THREAD

**Issue by de-served** [dragokas/hijackthis#234]
Не знаю, баг ли это, но...
понаотмечал галочек, не добавлял в игнорирование, чтобы сделать скриншоты для прошлых issue, пошёл для этого в т.ч. в Настройки, сделал скриншот, и... не нашёл способа вернуться к списку... капец... а я там рулон наотмечал без добавления в игнор-лист. :(((

**Comment by dragokas** [comment]
По нажатию на кнопку "Настройки", эта же кнопка сменяется на "Отчёт".
![image](https://github.com/dragokas/hijackthis/assets/19956568/a6bdd7fb-d25b-4f8a-b39e-629f53738d33)

В главном меню да, этой кнопки нет. Раньше была, но попадала под "оптимизацию".
Возможно, будет удобнее если кнопку "Только проверить систему" переименовать в "Результаты проверки" в случае, когда проверка уже была выполнена.

**Comment by de-served** [comment]
Да, кнопка-таки переименовывается... но это настолько неочевидно и интуитивно неуловимо, что... осталось за бортом.
Ведь видишь, что там кнопка "Настройки", кликаешь и взгляд уже где-то в другом месте, полагая, что ты же кликаешь по "Настройки" и оно там так и осталось. Ха-ха. Конфуз.
Я бы так не делал. Либо менял цвет кнопки, чтобы привлекала внимание. Ну то дело такое, "не моё", совать свои тапки в чужой огород не буду.
